src/Utilities/hooks/api/useReport.js

```diff
@@ -0,0 +1,7 @@
+import useQuery, { apiInstance } from '../useQuery';
+
+export const useReport = (id) => {
+  const query = useQuery(apiInstance.report, id);
```

Comment:
I think we do not have to store the result in another variable. Instead, we can directly return it.
```suggestion
  return useQuery(apiInstance.report, id);
```